ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/EpochCachePrimer.java

```diff
@@ -75,7 +75,9 @@ private void primeEpochStateCaches(final BeaconState state) {
 
     // Calculate committees for max lookahead epoch
     // (assume earlier epochs were already requested)
```

Comment:
the comment kind of suggests the change was consistent with the code, should probably update that too if maxLookahead is wrong...

Reply:
Good point, fixed.